tools/wpt/commands.json

```diff
@@ -5,10 +5,8 @@
     "parser": "create_parser",
     "help": "Run tests in a browser",
     "virtualenv": true,
-    "install": [
-      "zstandard"
-    ],
     "requirements": [
+      "requirements.txt",
```

Comment:
I definitely think the bits that only require `requests` shouldn't depend on `mozinstall` or `zstandard`. Ideally `mozinstall` wouldn't be needed unless you were actually trying to install Firefox. 

Generally, adding requirements when running e.g. Chrome seems like it could be a regression if you're using the wpt frontend internally but don't have access to all of pypi.

Reply:
I'll split things into more requirements files.